mingw-w64-xmake/PKGBUILD

```
@@ -27,6 +27,11 @@ prepare() {
 
 build() {
     cd "${_realname}-${pkgver}"
+
+    if [[ ${CARCH} == aarch64 ]]; then
+        export CFLAGS="-D_M_ARM64"
```

```suggestion
        export CFLAGS+=" -D_M_ARM64"
```

Or, something similar like that.